### ISSUE TITLE
[A11Y] All Editorial View - Settings area, the settings tabs are difficult to use and lack screenreader usability

### ISSUE THREAD

**Issue by Tribunal33** [pkp/pkp-lib#12720]
## Issue Description
Some of the auditors complained that the Settings are in general are difficult to navigate.  The tabs are not announcing the changes and arrow keys switch tabs instead of flowing through the setting options.

When describing the way the screenreader does not announce the settings unless focus mode is on they said:
> "To see all the settings, one must turn focus mode on and navigate the settings menu. Otherwise, the list of settings does not appear to NVDA. Within the settings, it can be confusing using NVDA as everything is a tab. One must turn focus mode on to change the tabs.

or 
> "To edit the Author guidance, one must click on the clickable application [area?] and then tab to the edit button[?].” “If the arrow keys are used in the format tool bar, only the bold italics and underline [words] are repeated by NVDA. 

_Note: There is no edit button in the Author Guidance area and the TinyMCE issue about not accessing format bar is addressed in another issue_

## How to reproduce / Current behavior
Login with Admin and head to Settings area.  In particular Workflow > Submission > Author Guidance

## What application are you using?
 OJS, OMP or OPS version 3.5.0-3

## Stack used 
- Device: Laptop
- OS: Windows 11
- Browser: Chrome
- Version: Latest

## Testing method
Manual using screenreader NVDA

## Action
Announce when tabs are being change and include ability to change the tabs without having focus mode turned on.  

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.
